frontend/src/metabase-types/api/dashboard.ts

```diff
@@ -18,7 +18,7 @@ export interface Dashboard {
   name: string;
   description: string | null;
   model?: string;
-  ordered_cards: (DashboardOrderedCard | ActionDashboardCard)[];
+  dashcards: (DashboardOrderedCard | ActionDashboardCard)[];
```

Comment:
I think we should also drop the word `Ordered` from:
- `DashboardOrderedCard`
- `BaseDashboardOrderedCard`
- `createMockDashboardOrderedCard`

Reply:
Yes, thank you!